projects/core/components/primitive-textfield/primitive-textfield.template.html

```diff
@@ -22,7 +22,7 @@
         tuiMaskAccessor
         automation-id="tui-primitive-textfield__native-input"
         class="t-input"
-        [attr.disabled]="computedDisabled || null"
+        [disabled]="computedDisabled"
```

Comment:
Should we also change this line ?

https://github.com/Tinkoff/taiga-ui/blob/f289acd28574f9b3d65c2c9deebd812c26c8e26c/projects/core/components/primitive-textfield/textfield/textfield.component.ts#L26

Reply:
Back in 2020 @MarsiBarsi changed it from property to attribute because exactly that bug was reproducible with the label. I'm not sure about circumstances, maybe it was Safari only, I have a vague memory of input using TextMask too. Our internal tracker does not have any meaningful description for that bug.

Reply:
Slider's disabled state is broken in Angular 15+ too 😭 

https://stackblitz.com/edit/disabled-slider-12-angular?file=src%2Fapp%2Fapp.component.ts
https://stackblitz.com/edit/disabled-slider-15-angular?file=src%2Fmain.ts

Reply:
I don't know why but we can't just change this line
https://github.com/Tinkoff/taiga-ui/blob/f289acd28574f9b3d65c2c9deebd812c26c8e26c/projects/kit/components/input-slider/input-slider.template.html#L39

to 
```html
[disabled]="readOnly || computedDisabled"
```

It does not work 🤔 
I will create the separate issue for sliders. It needs more investigation.

Reply:
@nsbarsukov no, It broken in ng15+ only with [ngModel] and [attr.disabled] together. In input[tuiTextfield] and range/slider everything should be ok, they have only [attr.disabled]

Reply:
I'll check input-slider test 🤔

Reply:
@vladimirpotekhin 
https://github.com/Tinkoff/taiga-ui/blob/f289acd28574f9b3d65c2c9deebd812c26c8e26c/projects/kit/components/input-slider/input-slider.template.html#L39-L40

https://github.com/Tinkoff/taiga-ui/blob/f289acd28574f9b3d65c2c9deebd812c26c8e26c/projects/kit/components/range/range.template.html#L30-L34

Reply:
😱 my bad, I didn't notice ngModel in slider. I'll change it too (angular 15+ just clears "disabled" attribute)

Reply:
> Back in 2020 @MarsiBarsi changed it from property to attribute because exactly that bug was reproducible with the label. I'm not sure about circumstances, maybe it was Safari only, I have a vague memory of input using TextMask too. Our internal tracker does not have any meaningful description for that bug.

I think the mask was the culprit. Tried to fix